docs/manifest.json

```diff
@@ -1,30 +1,40 @@
 {
   "versions": ["0.6.2"],
   "routes": [
+    {
+      "title": "Home",
+      "description": "Learn how to install and run Coder.",
```

Comment:
```suggestion
      "description": "Quickstart guide for using Coder.",
```

Reply:
The README is more quickstart-ish, so I suggest "How to install and run Coder" as a simplification

Reply:
Fixed